protocols/identify/src/behaviour.rs

```diff
@@ -63,18 +61,10 @@ pub struct Behaviour {
 }
 
 /// A pending reply to an inbound identification request.
-enum Reply {
-    /// The reply is queued for sending.
-    Queued {
-        peer: PeerId,
-        io: ReplySubstream<NegotiatedSubstream>,
-        observed: Multiaddr,
-    },
-    /// The reply is being sent.
-    Sending {
-        peer: PeerId,
-        io: Pin<Box<dyn Future<Output = Result<(), UpgradeError>> + Send>>,
-    },
+struct Request {
+    peer: PeerId,
```

Comment:
The handler could learn this via `IntoConnectionHandler`. It will never change across its lifespan.

Reply:
updated!